boards/arm/pan1783_pan1783a_evb/cpuapp_common.dtsi

```diff
@@ -3,7 +3,7 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#include "pan1783_evb_cpuapp_common-pinctrl.dtsi"
+#include "cpuapp_common-pinctrl.dtsi"
```

Comment:
let's keep the board in all the filenames (this applies to all includes and filenames) so that this requires the board prefix to be included and won't be wrongly included if someone has a same named file elsewhere, won't comment on others but please address those too